web/main.js

```diff
@@ -39,13 +39,49 @@ const getTime = (daysAgo) =>
   daysAgo >= 365 ? Math.floor(daysAgo / 365) + " years" : daysAgo + " days";
 
 const renderHeader = (ctnr) => {
-  const template = document.querySelector("#header-template").innerHTML;
-  const data = {
-    numGames: Object.values(metadata).length,
-    gameList: Object.values(metadata).map(_ => _.game_name).join(", "),
-  };
-  const html = applyDataToTemplate(data, template);
-  ctnr.insertAdjacentHTML("beforeend", html);
+  { // Container
+    const template = document.querySelector("#header-container-template").innerHTML;
+    const html = applyDataToTemplate({}, template);
```

Comment:
If there's no data I think you can skip this `applyDataToTemplate` line and render the template directly.

In fact, there doesn't need to be a template at all, but you may want to keep it if you don't want it to appear until after loading completes.